test/acceptance/tests.py

```diff
@@ -1100,7 +1109,6 @@ def test_train_self_peer_staff(self, peer_grades_me):
         # Do staff assessment step
         self.staff_assessment(peer_grades_me)
 
-
```

Comment:
This line shouldn't have been deleted - there ought to be 2 blank lines between classes.